gemfiles/rails5.2.gemfile

```diff
@@ -1,5 +1,4 @@
 eval_gemfile(File.expand_path('common.rb', __dir__))
 
-gem 'activerecord',  '>= 5.2.0.alpha', github: 'rails/rails', branch: 'master'
-gem 'activesupport', '>= 5.2.0.alpha', github: 'rails/rails', branch: 'master'
-gem 'arel',          '>= 9.0.0.alpha', github: 'rails/arel',  branch: 'master'
+gem 'activerecord',  '>= 5.2.0.beta'
+gem 'activesupport', '>= 5.2.0.beta'
```

Comment:
I’m not sure `activesupport` is needed. It’s not listed in the other gemfiles.

Reply:
Good call